api/v1/manager_types.go

```diff
@@ -29,7 +29,7 @@ type ManagerSpec struct {
 
 // ManagerStatus defines the observed state of the Calico Enterprise manager GUI.
 type ManagerStatus struct {
-	// The last successfully applied authentication configuration.
+	// Deprecated. Please use the Authentication CR for configuring authentication.
```

Comment:
We need some form of status to be here in order to meet openshift certification critera. 

if this one is deprecated we'll need to introduce something else - maybe just a "Ready" field like the others?

Reply:
Do you mean to do it like this?
```
// IntrusionDetectionStatus defines the observed state of Tigera intrusion detection capabilities.
type IntrusionDetectionStatus struct {
	// State provides user-readable status.
	State string `json:"state,omitempty"`
}
```

I see that while the ManagerStatus.Auth is deprecated here, the Manager.Spec.Auth is not. I should rectify this.

Reply:
> Do you mean to do it like this?

Yeah, all those resources that just have "State" is a way of meeting this requirement.

And yeah, we should have the spec and corresponding state field be the same deprecation status :)